cmd/snap/cmd_quota.go

```diff
@@ -45,24 +44,47 @@ The quotas command shows all quota groups.
 `)
 
 var shortRemoveQuotaHelp = i18n.G("Remove quota group")
-var longRemoveQuotaHelp = i18n.G(`The remove-quota command removes the given quota group.`)
+var longRemoveQuotaHelp = i18n.G(`
+The remove-quota command removes the given quota group. 
 
-type cmdQuota struct {
-	clientMixin
-	// MaxMemory and MemoryMax are mutually exclusive and provided for
-	// convienience, they mean the same.
-	MaxMemory  string `long:"max-memory" optional:"true"`
-	MemoryMax  string `long:"memory-max" optional:"true"`
-	Parent     string `long:"parent" optional:"true"`
-	Positional struct {
-		GroupName string              `positional-arg-name:"<group-name>" required:"true"`
-		Snaps     []installedSnapName `positional-arg-name:"<snap>" optional:"true"`
-	} `positional-args:"yes"`
-}
+Currently, only quota groups with no sub-groups can be removed. In order to 
+remove a quota group with sub-groups, the sub-groups must first be removed until
+there are no sub-groups for the group, then the group itself can be removed.
+`)
+
+var shortSetQuotaHelp = i18n.G(`Create or update a quota group.`)
+var longSetQuotaHelp = i18n.G(`
+The set-quota command updates or creates a quota group with the specified set of
+snaps.
+
+A quota group sets resource limits (currently maximum memory only) on the set of
+snaps that belong to it. Snaps can be at most in one quota group. Quota groups
+can be nested.
+
+All snaps provided are appended to the group; to remove a snap from a
+quota group the entire group must be removed with remove-quota and recreated 
+without the quota group. To remove a sub-group from the quota group, the 
```

Comment:
"without the quota group" -> "without the undesired snap"

Unless, of course, I misunderstood something :-)

Reply:
Yes good catch this was a typo. I will adjust the wording but not in this PR since it is finally green and I would love to land it :-)

Also this reminds me when @degville is back it would be great to have him look at the help texts here as well.